builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -359,12 +396,12 @@ def handleFailureEvent(errorMessage){
  */
 def unDeployService(stage) {
 	withCredentials([
-		[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: configLoader.AWS_CREDENTIAL_ID, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+		[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: service_config.credentialId, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
 	]) {
 		try {
 			def randomString = utilModule.generateRequestId();
 			def credsId = "jazz-${randomString}";
-			sh "aws configure set profile.${credsId}.region ${configLoader.AWS.REGION}"
```

Comment:
Do we need setting profile again within `withCredentials` context? I see that there are places where we do both & just `withCredentials` alone. Any specific reason for this?

Reply:
There's no specific reason. In some places, the credentials are being set as they need to be sent along to another helper function that requires them. I am waiting for mulit-account to be stable, before proposing the removal of all profile setting from the code